infrastructure/src/main/java/org/corfudb/infrastructure/ManagementServer.java

```diff
@@ -72,7 +72,8 @@
      * In milliseconds.
      */
     @Getter
-    private final long policyExecuteInterval = 1000;
+    @Setter
+    private volatile long policyExecuteInterval = 1000;
```

Comment:
Hm, why does this field need to be volatile? Is there a reason we would want to set this field at runtime rather than when ManagementServer is constructed?